addons/compute_worker/gpu_uniforms/gpu_uniform.gd

```diff
@@ -195,9 +240,17 @@ func byte_array_to_vec3_array(bytes: PackedByteArray) -> PackedVector3Array:
 func float_array_to_byte_array_64(array: Array[float]) -> PackedByteArray:
 	var bytes = PackedFloat64Array(array).to_byte_array()
 	return bytes
-	
+
+## Convert an array of Floats to GLSL equivalent `double[]`
+func int_array_to_byte_array_32(array: Array[int]) -> PackedByteArray:
+	var bytes = PackedInt32Array(array).to_byte_array()
+	return bytes
 
 ## Convert a GLSL `double[]` to an Array of Floats
 func byte_array_64_to_float_array(array: PackedByteArray) -> Array[float]:
 	return Array(array.to_float64_array())
 
+## Convert a GLSL `int[]` to an Array of Ints
+func byte_array_32_to_int_array(array: PackedByteArray) -> Array[float]:
```

Comment:
this function returns an Array[int] rather than Array[float]

Reply:
I'll resolve this tmrw, alongside other type specifications